benchmark/perf-tests/1000-rayjob/config.yaml

```diff
@@ -0,0 +1,82 @@
+name: kuberay
+namespace:
+  number: 100
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+steps:
+- name: Start measurements
+  measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: app.kubernetes.io/created-by = kuberay-operator
+      threshold: 30m
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: ray.io/v1
+      kind: RayCluster
+      labelSelector: app.kubernetes.io/created-by = kuberay-operator
+      operationTimeout: 120s
+  - Identifier: JobLifecycleLatency
```

Comment:
What does `JobLifecycleLatency` mean? Does it refer to the submitter K8s Job?

Reply:
yes, it's measuring time for Job to complete. In our case it measures submitter job time to completion, which roughly represents how long the training job took to complete